MovieKuno/Tab Pages/Home Views/MovieCell.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  MovieCell.swift
+//  MovieKuno
+//
+//  Created by Leah Joy Ylaya on 1/21/21.
+//
+
+import Foundation
+import SwiftUI
+
+struct MovieCell: View {
+    var movie: Movie
+    var body: some View {
+        VStack {
+            Image(Asset.moviePlaceholder.name)
+                .resizable()
+                .aspectRatio(contentMode: .fill)
+                .frame(width: 130, height: 200)
+                .cornerRadius(20)
+            Text(movie.title)
+                .modifier(Title(minHeight: 35, fontSize: 17))
```

Comment:
Better for modifier not having a parameters since its properties are defined inside?